src/config.cpp

```diff
@@ -311,6 +311,9 @@ void parse_program_options(int argc, const char** argv)
                 ->value_name("N")
                 ->default_value(11010113),
             "Sampling period (in number of events specified by -e).")
+        ("enable-system-sampling",
+            po::bool_switch(&config.system_mode_sampling),
+            "Enable sampling in system mode")
```

Comment:
The description reads `Enable sampling in system mode`, but a check (which I'd like more) or implicit enabling of system monitoring isn't done.
Aka. this does not fail with an error:
```
$ lo2s --enable-system-sampling
```